src/components/TextButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, ButtonProps, Space } from 'antd'
+import { RightCircleOutlined } from '@ant-design/icons'
+import { useContext } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+
+export function TextButton({ style, ...props }: ButtonProps) {
```

Comment:
Smol change.

We use this TextButton for the `ASSETS` button, and now the `HOLDERS` button